src/Raven.Server/Raven.Server.csproj

```diff
@@ -146,6 +146,13 @@
     <PackageReference Include="NuGet.PackageManagement" Version="5.11.0" />
     <PackageReference Include="NuGet.Protocol" Version="5.11.0" />
     <PackageReference Include="Oracle.ManagedDataAccess.Core" Version="3.21.3" />
+    <PackageReference Include="NEST" Version="7.13.0" />
+    <PackageReference Include="Npgsql" Version="5.0.5" />
+    <PackageReference Include="NuGet.ProjectModel" Version="5.8.1" />
+    <PackageReference Include="NuGet.Resolver" Version="5.8.1" />
+    <PackageReference Include="NuGet.PackageManagement" Version="5.8.1" />
+    <PackageReference Include="NuGet.Protocol" Version="5.8.1" />
+    <PackageReference Include="Oracle.ManagedDataAccess.Core" Version="3.21.1" />
```

Comment:
I think we only need one like here:

`<PackageReference Include="NEST" Version="7.13.0" />`

Reply:
Note that the rest is already specified above

Reply:
done